packages/waci-pex/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@sphereon/waci-pex",
+  "version": "0.0.1",
+  "description": "Veramo SSI QR Code provider",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "veramo": {
+    "pluginInterfaces": {
+      "ISsiQrCodeProvider": "./src/types/ISsiQrCodeProvider.ts"
+    }
+  },
+  "scripts": {
+    "build": "tsc --build"
+  },
+  "dependencies": {
+    "@sphereon/did-auth-siop": "^0.2.3",
+    "@veramo/core": "3.1.1-next.26",
+    "cross-fetch": "^3.1.4",
```

Comment:
3.1.5 is latest version

Reply:
Would have downloaded the version as ^ was used. But updated